ts/Series/Treemap/TreemapSeriesDefaults.ts

```diff
@@ -542,7 +558,120 @@ const TreemapSeriesDefaults: TreemapSeriesOptions = {
      *
      * @product highcharts
      */
-    traverseToLeaf: false
+    traverseToLeaf: false,
+
+    /**
+     * An option to optimize treemap series rendering by grouping smaller leaf
+     * nodes below a certain square area threshold in pixels. If the square area
+     * of a point becomes smaller than the specified threshold, determined by
+     * the `pixelWidth` and/or `pixelHeight` options, then this point is moved
+     * into group point per series.
```

Comment:
Did you mean into _one_ group point?
```suggestion
     * into one group point per series.
```